cmd/snap/main_test.go

```diff
@@ -80,6 +82,8 @@ func (s *BaseSnapSuite) RedirectClientToTestServer(handler func(http.ResponseWri
 
 type SnapSuite struct {
 	BaseSnapSuite
+
+	AuthFile string
```

Comment:
I'm pretty sure we really do need to extend these to all tests in the SnapSuite. At the moment, if you do anything related to login (including just run the tests normally), they do things like hoover up your real auth.json, or would overwrite it if you tried to log in in a test.